content/sensu-core/0.29/reference/handlers.md

```diff
@@ -186,7 +186,7 @@ Handler set definitions allow groups of handlers (i.e. individual collections of
 actions to take on event data) to be referenced via a single named handler set.
 
 _NOTE: Attributes defined on handler sets do not apply to the handlers they
-include. For example, `filter`, `filters`, and `mutator` attributes defined 
```

Comment:
I don't think we'll be removing `filter` from Sensu Core 1.x, so let's keep `filter` mentions in the handler reference.